### ISSUE TITLE
Anterior ankle osteophyte

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11215]
**Preferred term label:**
Anterior ankle osteophyte

**Synonyms**

**Definition (free text, please give PubMed ID)**
Imaging study of the ankle revealing osteophyte (bone spur) formation at the anterior aspect of the ankle (tibial or talar neck). This may be associated with tibiotalar synovitis, capsular thickening, scar, and granulation tissue as well; tibiotalar spur

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Anterior ankle impingement (tibiotalar impingement)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001332
- label: Anterior ankle osteophyte
- parents: [Abnormal ankle morphology (HP:0034673)](https://hpo.jax.org/app/browse/term/HP:0034673)
- pmid: PMID:22403038
- def: Imaging study of the ankle revealing osteophyte (bone spur) formation at the anterior aspect of the ankle (tibial or talar neck).
- comment: This finding may be associated with tibiotalar synovitis, capsular thickening, scar, and granulation tissue as well.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11215
- synonyms: Tibiotalar spur(NONE)
